dashboard/config/bits-kcsan.config

```diff
@@ -20,6 +22,9 @@ CONFIG_KCSAN_SKIP_WATCH_RANDOMIZE=y
 CONFIG_KCSAN_REPORT_VALUE_CHANGE_ONLY=y
 CONFIG_KCSAN_IGNORE_ATOMICS=y
 
+# Because KCSAN inserts delays, be more generous with RCU stall timeout.
+CONFIG_RCU_CPU_STALL_TIMEOUT=150
```

Comment:
It's not that simple, you can't just shuffle this single number. See:
https://github.com/google/syzkaller/commit/d3b2a0e212079
https://github.com/google/syzkaller/commit/809256c36c54e

Reply:
But does it really slower than KASAN+KUBSAN+a dozen of debug configs below? It seems wrong if it's slower.

Reply:
Hmm, good point. The question is if https://syzkaller.appspot.com/bug?extid=29a17bbebfece18b5ab9 is a genuine stall, or not.

1. If we just got unlucky that at the time where the CPU was IPI'd we were in KCSAN's runtime, then we can close this PR.

2. If it's not a genuine stall, e.g. caused by KCSAN setting up lots of watchpoints because we're stuck in a tight loop, then we ought to increase the delay.

Let me rethink this. If the stall doesn't reoccur (inside the runtime), we can probably declare it to be (1).

Reply:
As I noted we have a bunch of stalls/hangs with KASAN. 1 stall does not mean anything. However we filter out all stalls/hangs on KCSAN instance. This wasn't filtered out just because it happened in kcsan. So the question is: how many are there on the KCSAN instance? If no more than on KASAN instances, then I guess we just do nothing.
All these stalls/hangs may be a true kernel bug in the end, but it's mostly orthogonal to the problem at hand.

Reply:
If KCSAN is really noticeably slower, then this falls into #1552. There are lots of timeouts throughout.